### ISSUE TITLE
Refactor database repositories

### ISSUE THREAD

**Issue by vitaly-t** [LiskHQ/lisk#1376]
This can only be done after #1372 has been successfully merged and closed. ~Parent: #1306~

---

Refactor all database repositories to use:

* External SQL where possible
* ES6 classes syntax

PR-s here will not be updating issues pertaining to individual repository methods, such as:

* SQL semantics and usability
* Use of Named Parameters vs Index Variables
* Individual SQL documentation
* Individual method documentation
* Use of correct query methods

This is to be done with one PR per repository, hence a separate task for each:

 - [x] refactor repository `blocks` #1390 
 - [x] refactor repository `accounts`
 - [x] refactor repository `multisignatures` #1422 
 - [x] refactor repository `rounds` #1393 
 - [x] refactor repository `peers` #1419
 - [x] refactor repository `dapps` #1407
 - [x] refactor repository `votes` #1405
 - [x] refactor repository `voters` #1405

Note that repositories `migrations` and `transactions` are excluded here. They are both special cases, and there will be a separate issue open for each.

**Comment by vitaly-t** [comment]
Started with blocks, which turns out to be the biggest and the baddest of them all 😅

**Comment by vitaly-t** [comment]
Putting this task on hold, as the last ones - `accounts`, cannot be refactored, because of the open PR that's hanging around it for 12 days now: [Decouple SQL generation from account logic - Closes #1175](https://github.com/LiskHQ/lisk/pull/1312).

**Comment by vitaly-t** [comment]
Accounts refactoring done with [PR-1443](https://github.com/LiskHQ/lisk/pull/1443).

The boat is full, time to close it! :smile: It was good ride! :smile:
